packages/datatrak-web/src/Routes.tsx

```diff
@@ -4,14 +4,33 @@
  */
 
 import React from 'react';
-import { Route, Routes as RouterRoutes } from 'react-router-dom';
-import {
-  MainPageLayout,
-  SurveyPageLayout,
-  LandingPage,
-  SurveyPage,
-  SurveyQuestionsPage,
-} from './views';
+import { Navigate, Route, Routes as RouterRoutes } from 'react-router-dom';
+import { LandingPage, SurveyPage, SurveyQuestionsPage, LoginPage } from './views';
+import { useUser } from './api/queries';
+import { FullPageLoader } from '@tupaia/ui-components';
+import { ROUTES } from './constants';
+import { BackgroundPageLayout, MainPageLayout } from './layout';
+import { VerifyEmailPage } from './views/VerifyEmailPage';
+
+/**
+ * If the user is not logged in, redirect to the login page
+ */
+const LandingPageRedirect = () => {
+  const { isLoggedIn, isLoading, isFetched } = useUser();
+  if (isLoading || !isFetched) return <FullPageLoader />;
+  if (!isLoggedIn) return <Navigate to="/login" replace={true} />;
+  return <LandingPage />;
+};
+
+/**
+ * If the user is logged in and tries to access the login page, redirect to the home page
+ */
+const LoginRedirect = () => {
+  const { isLoggedIn, isLoading, isFetched } = useUser();
+  if (isLoading || !isFetched) return <FullPageLoader />;
+  if (isLoggedIn) return <Navigate to="/" replace={true} />;
+  return <LoginPage />;
+};
```

Comment:
```suggestion
const LoginRedirect = ({ children ) => {
  const { isLoggedIn, isLoading, isFetched } = useUser();
  if (isLoading || !isFetched) return <FullPageLoader />;
  if (isLoggedIn) return <Navigate to="/" replace={true} />;
  return children;
};
```
Can we either rename this or make it accept children. I was looking around for the login route in the Routes below and didn't realise it was up here.